src/components/Modal/Modal.tsx

```diff
@@ -34,6 +34,9 @@ export interface ModalProps {
    * for accessibility reasons.
    */
   onDismiss?: (event?: React.SyntheticEvent) => void;
+
+  /**  The z-index to be applied to the modal overlay. Overrides the default value of 1000  */
```

Comment:
This way our docs pick up the default value 😉 
```suggestion
  /**  
     * The z-index to be applied to the modal overlay  
     * @default 1000
     */
```